eth/backend.go

```diff
@@ -219,6 +219,10 @@ func New(stack *node.Node, config *Config) (*Ethereum, error) {
 		newBlockChainFunc = core.NewMultitenantBlockChain
 	}
 	eth.blockchain, err = newBlockChainFunc(chainDb, cacheConfig, chainConfig, eth.engine, vmConfig, eth.shouldPreserve, &config.TxLookupLimit)
+
+	// Quorum
+	eth.blockchain.SetSaveRevertReason(config.SaveRevertReason)
```

Comment:
Have you considered the option of updating `core.NewBlockChain` and `core.NewMultitenantBlockChain` to take `config` as input? (setting configuration after creating object may be error-prone)

Reply:
yes, I have and agree that is error-prone. the issue is that we need to change 100+ files, touching in many places of geth (because of `core.NewBlockChain`). one approach is to create a new object we call Quorum config or something so we can in the future pass other configuration needed (to avoid changing again 100+ files) @nmvalera wdyt?

Reply:
created https://github.com/ConsenSysQuorum/quorum/issues/253